File: TypescriptExample/Assignments/Product.ts

class Product{
    pId:number
    pName:string
    price:number
 
    constructor( pId:number, pName:string, price:number){
        this.pId=pId
        this.pName=pName
        this.price=price
    }
    addGst():number{
        return this.price*1.18
    }
    display():void{
        console.log(this.pId+" "+this.pName+" "+this.addGst())
        
    }
}
var p1=new Product(1,'bat', 100);
var p2=new Product(2,'ball', 200);
p1.display();
p2.display();
 

File: SpringBootExamples/SpringBeanProject/src/main/java/com/sonata/Product/Product.java
package com.sonata.Product;

public class Product {
private int pId;
private String pName;
private double pPrice;

 @Override
public String toString() {
	return "Product [pId=" + pId + ", pName=" + pName + ", pPrice=" + pPrice + "]";
}

Product(int pid,String pName,double price){
	 super();
	 this.pId=pid;
	 this.pPrice=price;
	 this.pName=pName;
 }
}
